promgen/templates/promgen/service_action_button_group.html

```diff
@@ -0,0 +1,49 @@
+{% load i18n %}
+{% load promgen %}
+
+
+<div style="display: inline-block;">
+  <div class="btn-group btn-group-sm" role="group" aria-label="...">
+    <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      {% translate "Actions" %} <span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu">
+
+      <li>
+        <a href="">
+          <form action="{% url 'service-notifier' service.id %}" style="display:inline" method="post" v-pre>{% csrf_token %}
+            <input type="hidden" name="sender" value="promgen.notification.user">
+            <input type="hidden" name="value" value="{{request.user.username}}" />
+            <button style="background:none;border:none;padding:0;">{% translate "Subscribe to Notifications" %}</button>
+          </form>
+        </a>
```

Comment:
Wrapping a form with an anchor tag feels hacky, especially with the href set this way (should probably be `href="#"` if it's a 'null' link. Even then, the button inline styles also feel hacky. What if we remove that and move it to a CSS rule ?

```css
.dropdown-menu > li > form > button {
  padding: 3px 20px;
  white-space: nowrap;
  background: none;
  border: none;
}
```

This was me figuring out the minimum to copy from the bootstrap 3 rules.

```suggestion
          <form action="{% url 'service-notifier' service.id %}" style="display:inline" method="post" v-pre>{% csrf_token %}
            <input type="hidden" name="sender" value="promgen.notification.user">
            <input type="hidden" name="value" value="{{request.user.username}}" />
            <button>{% translate "Subscribe to Notifications" %}</button>
          </form>
```

Or we can add a custom class to get around things.